### PR TITLE
fix-init-properties-script: fix init-properties script

### PR DESCRIPTION
**Pull request by yogsototh**

### CODE REVIEW

File: scripts/init-properties-for-docker.sh [hunk 1]
```diff
@@ -21,7 +21,7 @@ ctia.store.es.default.host=$ip
 ctia.store.es.default.port=9200
 ctia.hook.es.host=$ip
 ctia.hook.es.port=9200
-ctia.hook.redis.uri=http://${ip}:6379
```
Comment (quoll) [inline scripts/init-properties-for-docker.sh]: The expression should be wrapped in double quotes. So if there is ever an error that puts a space into the "ip" var, you don't accidentally set the uri to only include the first thing, and then get an error as it tries to execute the next term

Comment (yogsototh) [inline scripts/init-properties-for-docker.sh]: It should be good now.

File: scripts/init-properties-for-docker.sh [hunk 2]
```diff
@@ -17,11 +17,11 @@ if [ -f $resourcefile ]; then
 fi
 
 cat > $resourcefile <<EOF
-ctia.store.es.default.host=$ip
+ctia.store.es.default.host="$ip"
```
Comment (quoll) [inline scripts/init-properties-for-docker.sh]: I did not see the:
`cat > $resourcefile <<EOF`
in the previous context, so I did not realize it was going to a properties file, sorry. In that case you do not want quotes. But the move to host/port is preferable to URI.

Comment (yogsototh) [inline scripts/init-properties-for-docker.sh]: so back to previous version :).

The `uri` doesn't exists anymore, this is why I had to make this change, to use `host` instead.

**Pull request merged**